### ISSUE TITLE
update_translations.php can't be run directly

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#747]
**Reported by Nicolas on 21 Apr 38858549 16:00 UTC**
The script in `ops/update_translations.php` cannot be run directly as a program, you have to run it as '`php ops/update_translations.php`'. Patch:

```diff
Index: html/ops/update_translations.php
===================================================================
--- html/ops/update_translations.php (revision 16449)
+++ html/ops/update_translations.php (working copy)
@@ -1,3 +1,4 @@
+#!/usr/bin/env php
 <?php

 require_once("../inc/translation.inc");
@@ -4,4 +5,4 @@
 buildLanguages($lang_language_dir,$lang_translations_dir, $lang_compiled_dir);
 buildLanguages($lang_language_dir,$lang_prj_translations_dir, $lang_compiled_dir, true);

-?>
\ No newline at end of file
+?>
```
Additionally, set the svn:executable property on the file:
```bash
svn propset svn:executable yes html/ops/update_translations.php
```

**Comment by romw** [comment]
**Commented by davea on 4 Aug 39424754 11:06 UTC**
(In [18279]) - make update_translations.php executable; fixes #770

**Comment by romw** [comment]
**Commented by Nicolas on 25 Jun 39424762 09:46 UTC**
The change to the python scripts won't work, because the PHP script is not in the ```$PATH```. They should use ```./update_translations.php``` (or revert to calling the ```php``` command directly).

**Comment by romw** [comment]
**Commented by davea on 12 Nov 39424891 08:26 UTC**
(In [18282])  - fix python scripts; fixes #770 (from Nicolas)
